setup.py

```diff
@@ -89,6 +89,8 @@
     # Utilities from PyPA to e.g., compare versions
     "packaging",
     "responses<0.19",
+    # to populate metric template
+    "cookiecutter"
```

Comment:
do you want this to be a required dependency?

Reply:
moved it to a "template" requirement group.